tests/test_mesh_creation_functions.py

```diff
@@ -607,18 +678,53 @@ def test_mesh_creation_functions_element_length_option(
         create_beam_mesh_line(
             mesh,
             Beam3rHerm2Line3,
-            mat,
+            MaterialReissner(radius=0.1),
             [1.0, 2.0, 0.0],
             [3.0, 4.0, 6.0],
             n_el=1,
             l_el=l_el,
```

Comment:
```suggestion
            l_el=1.5,
```
and let's remove the previous definition of `l_el`.